packages/wonder-blocks-tooltip/components/tooltip.md

```diff
@@ -34,7 +34,7 @@ const {View, Text} = require("@khanacademy/wonder-blocks-core");
     <div style={{height: 100, overflow: "auto", border: "1px solid", margin: 10,}}>
         <div style={{height: "200vh"}}>
             <Text>This is a big long piece of text with a </Text>
```

Comment:
This is a place where we should definitely be using a typography style since we want to encourage devs to use them all the time.  Something like:
```
<Body>This is a big long piece of text with a 
   <Tooltip content={"Some long text"}>
         <Text style={{color: "red"}}>tooltip</Text>
   </Tooltip>
in the middle</Body>
```
I wonder if we could detect when `Tooltip`'s `children` is a `string` and automatically wrap it in a `Text` component.

Although this example shows off the flexibility of the tooltip it seems sub-optimal from an i18n perspective.

Reply:
We already are doing the detect if it's a string and wrap it thing.

Reply:
You commented on that in an earlier diff and I explained that's what it does.

I wouldn't stress on the tooltip docs just yet as they aren't really finished. The tooltip component is the last thing to be polished up.

Reply:
I just used the `Text` element to add a red-highlight. I can change it to anything we want; just wanted it to be clear where one should over.